wls-monitoring-service/src/main/java/de/muenchen/oss/wahllokalsystem/monitoringservice/client/waehleranzahl/WaehleranzahlClientMapper.java

```diff
@@ -0,0 +1,24 @@
+package de.muenchen.oss.wahllokalsystem.monitoringservice.client.waehleranzahl;
+
+import de.muenchen.oss.wahllokalsystem.monitoringservice.eai.aou.model.WahlbeteiligungsMeldungDTO;
+import de.muenchen.oss.wahllokalsystem.monitoringservice.service.waehleranzahl.WaehleranzahlModel;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper
+public interface WaehleranzahlClientMapper {
+
+    ZoneOffset DEFAULT_ZONE = ZoneOffset.UTC;
+
+    @Mapping(target = "wahlID", ignore = true)
+    @Mapping(target = "wahlbezirkID", source = "bezirkUndWahlID.wahlbezirkID")
+    @Mapping(target = "meldeZeitpunkt", source = "uhrzeit")
+    WahlbeteiligungsMeldungDTO fromModelToRemoteClientDTO(WaehleranzahlModel waehleranzahlModel);
+
+    default OffsetDateTime localDateTimeToOffsetDateTime(LocalDateTime localDateTime) {
+        return localDateTime.atOffset(DEFAULT_ZONE);
+    }
```

Comment:
Die zweite Methode ist notwendig um in den Tests zu Zeigen dass es mit Zeitzonen die eine Zeitumstellung haben funktioniert.

`zoneId.getRules().getOffset(localDateTime);` ist notwendig auf Grund der möglichen Zeitumstellung.

```suggestion
    default OffsetDateTime localDateTimeToOffsetDateTime(LocalDateTime localDateTime) {
        return localDateTimeToOffsetDateTime(localDateTime, DEFAULT_ZONE_ID);
    }

    default OffsetDateTime localDateTimeToOffsetDateTime(LocalDateTime localDateTime, ZoneId zoneId) {
        val zoneOffset = zoneId.getRules().getOffset(localDateTime);
        return localDateTime.atOffset(zoneOffset);
    }
```